contracts/modules/loan/Loan.t.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity 0.8.9;
+
+import { Escrow } from "../escrow/Escrow.sol";
+import { DSTest } from  "../../../lib/ds-test/src/test.sol";
+import { LoanLib } from "../../utils/LoanLib.sol";
+import { RevenueToken } from "../../mock/RevenueToken.sol";
+import { SimpleOracle } from "../../mock/SimpleOracle.sol";
+import { BasicEscrowedLoan } from "./BasicEscrowedLoan.sol";
+
+contract LoanTest is DSTest {
+
+    Escrow escrow;
+    RevenueToken supportedToken1;
+    RevenueToken supportedToken2;
+    RevenueToken unsupportedToken;
+    SimpleOracle oracle;
+    BasicEscrowedLoan loan;
+    uint mintAmount = 100 ether;
+    uint MAX_INT = 115792089237316195423570985008687907853269984665640564039457584007913129639935;
+    uint minCollateralRatio = 1 ether; // 100%
+
+    address borrower;
+    address arbiter;
+    address lender;
+
+    function setUp() public {
+        borrower = address(this);
+        lender = address(this);
+        arbiter = address(this);
+        supportedToken1 = new RevenueToken();
+        supportedToken2 = new RevenueToken();
+        unsupportedToken = new RevenueToken();
+        oracle = new SimpleOracle(address(supportedToken1), address(supportedToken2));
+        loan = new BasicEscrowedLoan(1 ether, address(oracle), arbiter, borrower, address(0), 1 ether);
+        escrow = loan.escrow();
+        _mintAndApprove();
+        escrow.addCollateral(1 ether, address(supportedToken1));
```

Comment:
You're using `supportedToken1` for all the loans so can you use `supportedToken2` for collateral? i dont think it affects current tests and i can't think of other tests to add for now but I can see that being a requirement for some tests in the future